src/trace/datadog-tracer.ts

```diff
@@ -0,0 +1,14 @@
+import Tracer from "dd-trace";
+
+const AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME";
+
+export function initDatadogTracer() {
+  const functionName = process.env[AWS_LAMBDA_FUNCTION_NAME];
+  const service = functionName ? `${functionName}` : "lambda";
```

Comment:
Is it a surprise if we need to default service name to `lambda` becasue function name is unavailable? If yes, we probably should log a warning to let user know? Otherwise, users will be super confused when this happens.

Reply:
Mostly for where the user is running in a test environment where that variable might not be defined. I'll log an error in that case.